### DISCUSSION TITLE
CSV output for composer show, composer licenses

### DISCUSSION THREAD

**Discussion by simonrjones** [composer/composer#12144] [General]
At my agency I work with larger organisations who require lists of third party packages used on a web application.

I have been compiling these lists via composer show (or composer licenses to get the license info too). I see the `--ouput` format supports text or json. It would be handy to support CSV output too so I can copy this list into Excel. 

I'd be happy to look at contributing code for this, but I wanted to ask if this is an appropriate feature to add to Composer first.

**Comment by Seldaek** [comment]
I think I'd rather you do script it yourself from the json output tbh. I don't really want to maintain a long list of formats.

For example using https://jqlang.github.io/jq/:

```
composer show --format=json | jq -r '.installed | (["name","version"], (.[] | [.name, .version])) | @csv'
"name","version"
"composer/ca-bundle","1.5.2"
"composer/class-map-generator","1.3.4"
"composer/metadata-minifier","1.0.0"
"composer/pcre","2.3.1"
"composer/semver","3.4.3"
"composer/spdx-licenses","1.5.8"
"composer/xdebug-handler","3.0.5"
....
```
